scipy/stats/_continuous_distns.py

```diff
@@ -2224,20 +2227,21 @@ def _entropy(self, a, c):
 
 
 class genhalflogistic_gen(rv_continuous):
-    """A generalized half-logistic continuous random variable.
+    r"""A generalized half-logistic continuous random variable.
 
     %(before_notes)s
 
     Notes
     -----
-    The probability density function for `genhalflogistic` is::
+    The probability density function for `genhalflogistic` is:
+
+    .. math::
 
-        genhalflogistic.pdf(x, c) =
-            2 * (1-c*x)**(1/c-1) / (1+(1-c*x)**(1/c))**2
+        f(x, c) = \frac{2 {(1-c x)}^{\frac{1}{c-1}}}{{\left (1+{(1-c x)}^{1/c} \right)}^2}
```

Comment:
I think mine looks much better in LaTeX. See

![Imgur](http://i.imgur.com/fnNAcVm.png)

the first one is mine, the second one is the LaTeX you suggest, the thirds is a mixture `\frac{2 {(1 - c x)}^{1/(c-1)}}{{\left[1 + {(1 - c x)}^{1/c} \right]}^2}`

I don't care about the type of brace. Using `\left` / `\right` makes ist easier to read. Enclosing the base of the exponent in curly braces allows LaTeX to recognize the term correctly (otherwise, LaTeX will only take the last character).

Anyway, I think all 3 options are much better than what we currently have.